Readme.md

```diff
@@ -24,6 +24,9 @@ https://dbdiagram.io/d/5d4b8094ced98361d6dd6837
 <dt><a href="#findById">findById(table, id)</a> ⇒</dt>
 <dd><p>Database model to get a single record by id</p>
 </dd>
+<dt><a href="#findByEmail">findByEmail(email)</a> ⇒</dt>
```

Comment:
Thanks for implementing this one. I was just thinking that we'd need it to look up users.